a3p-integration/proposals/l:wallet-upgrade/test/wf.test.js

```diff
@@ -0,0 +1,110 @@
+// @ts-check
+import '@endo/init/debug.js';
+
+import {
+  LOCAL_CONFIG,
+  makeVstorageKit,
+  retryUntilCondition,
+} from '@agoric/client-utils';
+import {
+  agoric,
+  getIncarnation,
+  GOV1ADDR,
+  mkTemp,
+} from '@agoric/synthetic-chain';
+import test from 'ava';
+import { writeFile } from 'node:fs/promises';
+
+/**
+ * @import {BridgeAction} from '@agoric/smart-wallet/src/smartWallet';
+ */
+
+test(`walletFactory incarnation reflects invokeEntry on existing wallets`, async t => {
+  const history = { walletFactory: 10 };
+  t.is(await getIncarnation('walletFactory'), history.walletFactory);
+});
+
+test('pre-existing wallets can use saveResultAs and invokeEntry', async t => {
```

Comment:
I don't see any successful use. We're taking error messages as enough evidence? hm.

I suppose the functionality is already tested elsewhere and here the bar is kinda low: just prove that we're talking to the relevant code.

ok.